API-source/routes/todos.php

```diff
@@ -53,24 +53,16 @@
   global $logged_in_user;
   $original_todolist_id = $todolist_id;
 
-  /* IF TEAMGANTT TODOLIST - UPDATE TOLIST_ID */
```

Comment:
Since we no longer have the 'teamgantt todolist' we don't need to track the todo how we used to. We can simply treat it like a normal todo & then if it's linked to TG, at the end of the update we can apply the patch.